lib/utils.js

```diff
@@ -7,6 +7,12 @@ function isPlainObject(value) {
 }
 exports.isPlainObject = isPlainObject;
 
+// Support dynamic require within webpack bundle.
+if (typeof __non_webpack_require__ === 'undefined') {
+  var __non_webpack_require__ = require;
```

Comment:
What I'm saying is that the `typeof` check isn't doing what it's supposed to, thanks to the `var` binding:
```js
var value = 123;
console.log(function() {
  if (typeof value === "undefined") {
    var value = 456;
  }
  return value;
}()); // logs 456
```